### PR TITLE
updating build.gradle to allow publishing SNAPSHOTS

### PR DESCRIPTION
**Pull request by lbergelson**
adding automatic versioning like the rest of our tools
adding a snapshot repository to publish to

### CODE REVIEW

**Comment by lbergelson** [comment]
@cmnbroad Could you take a look at this one?  The lack of build infrastructure in this repo is a pain.

File: build.gradle [hunk 1]
```diff
@@ -75,6 +78,10 @@ uploadArchives {
                 authentication(userName: project.findProperty("sonatypeUsername"), password: project.findProperty("sonatypePassword"))
             }
 
+            snapshotRepository(url: "https://broadinstitute.jfrog.io/broadinstitute/libs-snapshot-local/") {
+                authentication(userName: System.env.ARTIFACTORY_USERNAME, password: System.env.ARTIFACTORY_PASSWORD)
```
Comment (cmnbroad) [inline build.gradle:82]: @lbergelson Don't you also need the corresponding travis.yml entries for this ?

Comment (cmnbroad) [reply build.gradle:82]: Oh, are you just going to do this manually.

Comment (lbergelson) [reply build.gradle:82]: just to be able to do it manually.  I should add travis as well, but that's a different pr I think.

**Review by cmnbroad — APPROVED** [review]
:+1:

File: build.gradle (hunk 1 above)
Comment (cmnbroad) [reply build.gradle:82]: Oh wow - I see now. There is no travis CI at all for this repo.

**Pull request merged**